arangod/RestServer/DatabaseFeature.cpp

```diff
@@ -1384,3 +1393,65 @@ void DatabaseFeature::enableDeadlockDetection() {
     vocbase->_deadlockDetector.enabled(true);
   }
 }
+
+std::pair<std::string /*sharding*/, std::size_t /*replication*/>
+arangodb::getOneShardOptions(std::string const& collectionName, VPackSlice options) {
```

Comment:
let's split this into two separate methods, each of which should return exactly one thing.